frontend/src/components/RecentOrderCard/index.tsx

```diff
@@ -0,0 +1,167 @@
+import {Typography, styled, Box} from '@mui/material';
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent';
+import Chip from '@mui/material/Chip/Chip';
+import busIcon from '../../assets/bus-icon.svg';
+import arrowIcon from '../../assets/arrowIcon.svg';
+import scheduleIcon from '../../assets/schedule-icon.svg';
+
+export default function RecentOrderCard() {
```

Comment:
Let's create a new directory inside the components directory with the name `admin-panel` and move this component there for better management.